sematic/BUILD

```diff
@@ -102,6 +102,7 @@ sematic_py_lib(
     srcs = ["abstract_future.py"],
     deps = [
         ":abstract_calculator",
+        ":retry_settings",
```

Comment:
you need to add it as a dependency of `init` as well

Reply:
It's already there.